system/sharedClasses/playlist.xml

```diff
@@ -28,9 +28,9 @@
 			<variation>Mode110</variation>
 			<variation>Mode610</variation>
 		</variations>
-		<command>if [[ ! $(Q)$(PLATFORM)$(Q) == *$(Q)win$(Q)* ]] ; then export original_umask_val=`umask` $(AND_IF_SUCCESS) umask 0002 $(AND_IF_SUCCESS) echo umask set to `umask`; fi; \
+		<command>if [ ! $(Q)$(PLATFORM)$(Q) == *$(Q)win$(Q)* ] ; then export original_umask_val=`umask` $(AND_IF_SUCCESS) umask 0002 $(AND_IF_SUCCESS) echo umask set to `umask`; fi; \
```

Comment:
Why remove the double `[[`?

Reply:
Because that's a bash extension and the default shell is dash on a number of our test machines.  However I've also found that the `-u` option in `ulimit -u` is also absent in dash so I'm working on a solution to work round that too (probably by checking bash is installed!).
Found this while implementing @mesbah's 'why not just print the ulimit -a' output.
I'll remove the 'WIP:' when I think I've got a fix.